resource-watch/resource-watch
# Explore new layout

![image](https://user-images.githubusercontent.com/545342/75861038-1776eb80-5dfd-11ea-813a-1a3d3d0d3109.png)

![image](https://user-images.githubusercontent.com/545342/75861088-2a89bb80-5dfd-11ea-8c15-fea40d91b9d5.png)

## Overview
This PR includes a first approximation to the new Explore layout.
The following updates are included:
- New icons added to `Icomoon` and our icons component
- Jenkins file necessary updates for `explore` branch new domain
- Sidebar menu component
- Topics section including buttons according to the design prototype - an initial approach for filtering by them is also working.
- New Explore dataset cards fully functional
- Selected section stored in Redux and URL, having the interface to be consistently built when reloading the URL
among other things...

**NOTE**: bear in mind that there are many "unfinished" things so there might be some style issues that will be tackled in later PRs.

## Testing instructions
Go to http://localhost:9000/data/explore and check the `All data` and `Topics` section.
